src/metabase/driver/postgres.clj

```diff
@@ -858,14 +859,20 @@
   [driver]
   (= driver :postgres))
 
+
 (defmethod sql-jdbc.sync/alter-columns-sql :postgres
   [driver table-name column-definitions]
-  (first (sql/format {:alter-table  (keyword table-name)
-                      :alter-column (map (fn [[column-name type-and-constraints]]
-                                           (vec (cons column-name (cons :type type-and-constraints))))
-                                         column-definitions)}
-                     :quoted true
-                     :dialect (sql.qp/quote-style driver))))
+  (with-quoting driver
+    (first (sql/format {:alter-table  (keyword table-name)
+                        :alter-column (map (fn [[column-name type-and-constraints]]
+                                             (vec (list* (quote-identifier column-name)
+                                                         :type
```

Comment:
Quite annoying to have an override for such a silly piece of syntax. Perhaps we want to use composition instead of overriding somehow for better future proofing next time.